src/Neo.CLI/Settings.cs

```diff
@@ -112,7 +112,7 @@ public UnlockWalletSettings(IConfigurationSection section)
             {
                 this.Path = section.GetValue("Path", "");
                 this.Password = section.GetValue("Password", "");
-                this.IsActive = bool.Parse(section.GetValue("IsActive", "false")!);
+                this.IsActive = bool.Parse(section.GetValue("IsActive", "false"));
```

Comment:
This kind of changes are needed?

Reply:
We just need to change to `section.GetValue<bool>("IsActive", false)`, ill fix later